pulp_python/app/models.py

```diff
@@ -157,6 +162,9 @@ class PythonRemote(Remote):
     prereleases = models.BooleanField(default=False)
     includes = JSONField(default=list)
     excludes = JSONField(default=list)
+    package_types = ArrayField(models.CharField(max_length=15, blank=True), choices=PACKAGE_TYPES, default=list)
+    latest_kept = models.IntegerField(default=0)
+    exclude_platforms = JSONField(default=list)
```

Comment:
I need some help with choosing what is the appropriate model to store the package_types and exclude_platforms info. I want to use the MultipleChoiceField in the serializers but both models have problems with that field converting the data correctly.